src/MiniProfiler.Shared/DiagnosticInitializer.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Threading;
```

Comment:
Why this using statement?

Reply:
Because I almost made it an `Interlocked` `int` swap :) (removing!)

Reply:
Ha! I figured. Since this should be called in startup, the fact that it's not thread should be fine. You and I both know there will be someone out there who will try to parallelize app startup and they'll be wrong, but not sure it makes sense to worry about that. 👍